callback_plugins/homsaplog.py

```diff
@@ -0,0 +1,71 @@
+
+from __future__ import (absolute_import, division, print_function)
```

Comment:
graag verplaatsen naar de andere imports.


"Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants."